app/controllers/comment_liking_relations_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CommentLikingRelationsController < ApplicationController
+  before_action :authenticate
+
+  def create
```

Comment:
czy w sumie tu nie musimy się zabezpieczyć, i sprawdzić czy już nie ma takiej pary dla comment_id i user_id?

Nie jestem pewien czy model tego nie sprawdza już jakoś

Reply:
Właściwie to dodałem to na poziomie bazy danych przy tworzeniu migracji. Dodałem indeks łączący lajkującego z komentarzem i dałem warunek, że nie może być duplikatów. A w aplikacji nie będzie opcji lajknąć dwa razy.

Migracja:
add_index :comment_liking_relations,
              [:liking_user_id, :liked_comment_id],
              unique: true,
              name: 'index_liking_user_and_comment'